R/cf.R

```diff
@@ -482,15 +670,48 @@ addStat.cf <- function(cf1, cf2) {
   stopifnot(inherits(cf1, 'cf_meta'))
   stopifnot(inherits(cf2, 'cf_meta'))
 
+  ##Either both should have an index or none of them
+  stopifnot(inherits(cf1, 'cf_indexed') == inherits(cf1, 'cf_indexed') )
+
   stopifnot(cf1$Time == cf2$Time)
   stopifnot(dim(cf1$cf)[2] == dim(cf2$cf)[2])
   stopifnot(cf1$nrObs == cf2$nrObs )
   stopifnot(cf1$nrStypes == cf2$nrStypes)
 
   cf <- cf1
 
-  cf$cf <- rbind(cf1$cf, cf2$cf)
-  cf$icf <- rbind(cf1$icf, cf2$icf)
+  cf1_temp<- cf1$cf
+  icf1_temp <- cf1$icf
+  if (reverse1 == TRUE){
+    apply(cf1_temp,2,rev)
+    if ( has_icf(cf1)){
+      str(cf1$icf)
```

Comment:
str should be removed here

Reply:
I think this is also has been removed

Reply:
but it's right there

Reply:
`str` is still there it seems..

Reply:
and this one?

Reply:
I think it is fixed, using commit 2718d0413cff68c3d573807ba92df066846ad7ed, i do not see this str line